### PR TITLE
chore: linting neps

### PR DESCRIPTION
**Pull request by robert-zaremba**
In https://github.com/near/NEPs/pull/472 we integrated a popular linter. Here we update the existing markdown files to clean up linting issues.

- [x] root dir
- [x] neps
- `specs`  -- let's do it in other PR, will be easier to review.

### CODE REVIEW

File: neps/nep-0141.md [hunk 1]
```diff
@@ -293,35 +295,38 @@ Finalize an `ft_transfer_call` chain of cross-contract calls.
...
 
- 1. Sender calls `ft_transfer_call` on FT contract
- 2. FT contract transfers `amount` tokens from sender to receiver
- 3. FT contract calls `ft_on_transfer` on receiver contract
- 4+. [receiver contract may make other cross-contract calls]
- N. FT contract resolves promise chain with `ft_resolve_transfer`, and may
+1.  Sender calls `ft_transfer_call` on FT contract
+2.  FT contract transfers `amount` tokens from sender to receiver
+3.  FT contract calls `ft_on_transfer` on receiver contract
+    4+. [receiver contract may make other cross-contract calls]
```
Comment (frol) [inline neps/nep-0141.md:301]: The formatting here and the next line will be broken. I guess we should just change it to 4. And 5.

File: neps/nep-0148.md [hunk 2]
```diff
@@ -35,34 +36,33 @@ A fungible token smart contract allows for discoverable properties. Some propert
...
+##### High-level explanation
 
 Alice issues a transaction to deploy and initialize the fungible token contract, providing arguments to the initialization function that set metadata fields.
 
-**Technical calls**
+##### Technical calls
 
 1. `alice` deploys a contract and calls `wbtc::new` with all metadata. If this deploy and initialization were done using [NEAR CLI](https://docs.near.org/tools/near-cli) the command would be:
 
-    near deploy wbtc --wasmFile res/ft.wasm --initFunction new --initArgs '{
```
Comment (frol) [inline neps/nep-0148.md:53]: Let's put it into fences as otherwise the indentation is ruined

File: neps/nep-0178.md [hunk 3]
```diff
@@ -57,133 +57,133 @@ Let's examine the technical calls through the following scenarios:
...
+          "msg": "{\"action\": \"list\", \"price\": \"100\", \"token\": \"nDAI\" }"
+        }' --accountId alice --depositYocto 1
 
-   At this point, near-cli will hang until the cross-contract call chain fully resolves, which would also be true if Alice used a Market frontend using [near-api-js](https://docs.near.org/tools/near-api-js/quick-reference). Alice's part is done, though. The rest happens behind the scenes.
+    At this point, near-cli will hang until the cross-contract call chain fully resolves, which would also be true if Alice used a Market frontend using [near-api-js](https://docs.near.org/tools/near-api-js/quick-reference). Alice's part is done, though. The rest happens behind the scenes.
 
-2. `nft` schedules a call to `nft_on_approve` on `market`. Using near-cli notation for easy cross-reference with the above, this would look like:
+2.  `nft` schedules a call to `nft_on_approve` on `market`. Using near-cli notation for easy cross-reference with the above, this would look like:
 
-       near call market nft_on_approve '{
```
Comment (frol) [inline neps/nep-0178.md:126]: Again, let's use fences block here to save proper integration in the command

**Review by frol — CHANGES_REQUESTED** [review]
This is great! I spotted a couple of formatting issues and once they are addressed, we are ready to merge

File: neps/nep-0148.md (hunk 2 above)
Comment (robert-zaremba) [reply neps/nep-0148.md:53]: In fact the linter suggests to use fences. However I disabled that check to reduce amount of modifications. However, it seams that auto-fix modified it anyway.

File: neps/nep-0141.md (hunk 1 above)
Comment (robert-zaremba) [reply neps/nep-0141.md:301]: Most of things were auto-formatted. Yes, will need to go through the diff.

File: neps/nep-0141.md [hunk 4]
```diff
@@ -132,8 +134,7 @@ Alice needs to issue 1 transaction, as opposed to 2 with a typical escrow workfl
 #### Swapping one token for another via an Automated Market Maker (AMM) like Uniswap
 
 Alice wants to swap 5 wrapped NEAR (wNEAR) for BNNA tokens at current market rate, with less than 2% slippage.
-
-**Assumptions**
+Assumptions
```
Comment (frol) [inline neps/nep-0141.md:137]:
```suggestion

##### Assumptions
```

File: neps/nep-0148.md [hunk 5]
```diff
@@ -35,34 +36,34 @@ A fungible token smart contract allows for discoverable properties. Some propert
 #### Token provides metadata upon deploy and initialization
 
 Alice deploys a wBTC fungible token contract.
-
-**Assumptions**
+Assumptions
```
Comment (frol) [inline neps/nep-0148.md:39]:
```suggestion

##### Assumptions
```

**Review by frol — APPROVED** [review]

File: neps/nep-0141.md [hunk 6]
```diff
@@ -63,33 +66,31 @@ Note that precision (the number of decimal places supported by a given token) is
...
 
 1. `alice` calls `wbtc::ft_transfer({"receiver_id": "bob", "amount": "500000000"})`.
 
-##### Token deposit to a contract
+#### Token deposit to a contract
 
 Alice wants to deposit 1000 DAI tokens to a compound interest contract to earn extra tokens.
-
-**Assumptions**
+Assumptions
```
[the remark this answers is not in the document]
Comment (robert-zaremba) [reply neps/nep-0141.md]: Actually, I intentionally didn't make "Assummptions" as a new header. IMHO, it looked better if we included it in the main flow of the use case description.

[the remark this answers is not in the document]
Comment (robert-zaremba) [reply neps/nep-0141.md]: it's ok, no need to change second time.

**Pull request merged**